### ISSUE TITLE
No traceback on error: TypeError: 'str' object does not support item assignment 

### ISSUE THREAD

**Issue by MrMrRobat** [dephell/dephell#190]
```bash
$ dephell deps convert --traceback --level=DEBUG --from=pyproject.toml --to pip --to-path dev-requirements.txt
ERROR TypeError: 'str' object does not support item assignment
```

**Config:**
```toml
[tool.poetry.dependencies]
python = ">=3.6"

[tool.poetry.dev-dependencies]
pytest = "*"

[tool.dephell.main]
from = {format = "poetry", path = "pyproject.toml"}
to = {format = "setuppy", path = "setup.py"}
```
`dephell inspect self` **output:**
```json
{
  "cache": "44.09Mb",
  "path": "/Users/rocky/dev/py/dephell/dephell",
  "python": "/Library/Frameworks/Python.framework/Versions/3.6/bin/python3",
  "version": "0.7.4",
  "versions": {}
}
```

**Comment by orsinium** [comment]
Is `dev-requirements.txt` already exists? If you delete `dev-requirements.txt`, will you have the same error?

**Comment by MrMrRobat** [comment]
Yes, it exists. No, if I delete it, issue remain the same.

**Comment by orsinium** [comment]
1. Please, use `--to-format` instead of `--to` here.
1. Missed traceback fixed in #202.

Thank you for your feedback :)
